src/Microsoft.ML.Transforms/Text/NgramHashingTransformer.cs

```diff
@@ -875,15 +875,30 @@ public sealed class NgramHashingEstimator : IEstimator<NgramHashingTransformer>
         /// </summary>
         public sealed class ColumnInfo
         {
+            /// <summary>Name of the column resulting from the transformation of <see cref="InputColumnNames"/>.</summary>
             public readonly string Name;
+            /// <summary>Names of the columns to transform.</summary>
             public readonly string[] InputColumnNames;
+            /// <summary>Maximum ngram length.</summary>
             public readonly int NgramLength;
+            /// <summary>Maximum number of tokens to skip when constructing an ngram.</summary>
             public readonly int SkipLength;
+            /// <summary>Whether to store all ngram lengths up to ngramLength, or only ngramLength.</summary>
             public readonly bool AllLengths;
+            /// <summary>Number of bits to hash into. Must be between 1 and 31, inclusive.</summary>
             public readonly int HashBits;
+            /// <summary>Hashing seed.</summary>
             public readonly uint Seed;
+            /// <summary>Whether the position of each term should be included in the hash.</summary>
             public readonly bool Ordered;
+            /// <summary>
+            /// During hashing we constuct mappings between original values and the produced hash values.
+            /// Text representation of original values are stored in the slot names of the  metadata for the new column.Hashing, as such, can map many initial values to one.
```

Comment:
>w column.Hashing,  [](start = 111, length = 18)

are you referring to column name "column.Hashing"? if so please put in quotes otherwise it might be you are missing a space before "Hashing.."

Reply:
Beyond just quotes, what is actually meant by `column.Hashing`? Is this referring to some sort of member of some class or struct somewhere? Maybe we casn consider adding proper `<see` tags to refer to whatever actual member is being discussed (which is very unclear to me from the context, so may be unclear to any reader.)

Reply:
Ok I will try to address your comments. This is a particularly hairy one and we had a few emails back and forth between Senja and Ivan when we did the conversion to estimator to try to make it more clear but I guess we can do better if both of you have comments.

---
In reply to: [258318584](https://github.com/dotnet/machinelearning/pull/2648#discussion_r258318584) [](ancestors = 258318584)

Reply:
I think most of the confusion was coming from the need of a space like Zeeshan pointed out. With the space the sentence reads much better. 

---
In reply to: [258628869](https://github.com/dotnet/machinelearning/pull/2648#discussion_r258628869) [](ancestors = 258628869,258318584)